Makefile

```diff
@@ -438,6 +438,7 @@ test: release
 	@$(MAKE) $(QUIET_MAKE) -C test default
 
 testall: release
+	cp $(build_prefix)/lib/julia/sys.ji local.ji && ./julia -J local.ji -e 'true' && rm local.ji
```

Comment:
This won't work on windows, where there's no `./julia` symlink. Use `$(JULIA_EXECUTABLE)`